### PR TITLE
Updating dependencies and fixing failing Ping Bell test

### PR DESCRIPTION
**Pull request by markpete**
* All dependencies have been updated to the latest versions
* Fixed regex to work on Windows, Linux, & Mac
* Fixed the Ping Bell solution and test to use play() instead of the deprecated tone().

### CODE REVIEW

**Comment by markpete** [comment]
Regarding the failing checks above.
**Node Security** - It's reporting a problem with a version of one of the sub-dependencies - 'hoek'.  This pre-dates my updates.
**CI Failures** - The build fails on Node 0.10 and 0.12, but passes on later versions.  I would suggest removing 0.x versions of node from the CI and adding a later 8 and 9.

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "nodebot-workshop",
-  "version": "3.2.0",
+  "version": "3.2.1",
```
Comment (achingbrain) [inline package.json]: Please remove this, the version will be bumped during the next `npm version && npm publish`

**Comment by markpete** [comment]
I removed the visual color check for Windows as the previous UX was unreadable.
Before:
windows menu
After:
windows menu - 220

File: package.json (hunk 1 above)
Comment (markpete) [reply package.json]: Reverted this part.

**Comment by achingbrain** [comment]
Published as v3.2.1.

I rebased your changes on top of master as it looks like the git history got a little mixed up when you squashed your commits.

Thanks so much for this!

**Comment by markpete** [comment]
Thanks!  Glad it was useful.

Sent from my iPhone

On Mar 3, 2018, at 3:43 AM, Alex Potsides <notifications@github.com<mailto:notifications@github.com>> wrote:


Closed #47<https://eur01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Ftableflip%2Fnodebot-workshop%2Fpull%2F47&data=02%7C01%7C%7Ccf570ecd187f43bfd1ee08d580fc07f6%7C84df9e7fe9f640afb435aaaaaaaaaaaa%7C1%7C0%7C636556742308495934&sdata=uuwikU%2Bn%2Fjv2Dx%2BiYjP3pId43ZZs1BzkeOx9WbCys7w%3D&reserved=0>.

**Pull request closed without merging**